src/functions-and-arrays.js

```diff
@@ -1,19 +1,80 @@
 // Iteration #1: Find the maximum
+function maxOfTwoNumbers(firstNum, secondNum){
+  if (firstNum > secondNum){
+    return firstNum;
+   } else if (firstNum < secondNum){
+       return secondNum;
+   } else {
+     return firstNum;
+   }
+}
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
+function findLongestWord(words){
+  var maxLength = 0;
+  var longestWord = '';
+
+  if(words.length == 0){
+    return null; 
+  }
+
+  for(var i = 0; i < words.length; i++){
+    if(words[i].length > maxLength){
+      maxLength = words[i].length;
+      longestWord = words[i];
+    }
+  }
+  return longestWord;
+}
+
 // Iteration #3: Calculate the sum
 
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+function sumNumbers(numbers){
+  var sum = 0;
+  for (var i = 0; i < numbers.length; i++){
+    sum += numbers[i];
+  }
+return sum;
```

Comment:
Please try to be mindful about the vertical formatting and the indentation here.